spec/requests/items_spec.rb

```diff
@@ -47,7 +47,18 @@
   describe 'GET /items/:id' do
     context 'when the record exists' do
       it 'returns the item' do
-        items = FactoryBot.create_list(:item, 10)
+        items = [
+          FactoryBot.create(:item, name: 'Raspberry', sell_in: 5, quality: 11),
+          FactoryBot.create(:item, name: 'Pineapple', sell_in: 6, quality: 12),
+          FactoryBot.create(:item, name: 'Basil', sell_in: 7, quality: 13),
+          FactoryBot.create(:item, name: 'Bananas', sell_in: 8, quality: 14),
+          FactoryBot.create(:item, name: 'Granola', sell_in: 9, quality: 15),
+          FactoryBot.create(:item, name: 'Foo', sell_in: 9, quality: 15),
+          FactoryBot.create(:item, name: 'Bar', sell_in: 16, quality: 26),
+          FactoryBot.create(:item, name: 'Lorem', sell_in: 9, quality: 19),
+          FactoryBot.create(:item, name: 'Ipsum', sell_in: 3, quality: 5),
+          FactoryBot.create(:item, name: 'Onions', sell_in: 8, quality: 15)
+        ]
         item_id = items.first.id
 
         get "/api/v1/items/#{item_id}"
```

Comment:
Question: What are you asserting here? What is being tested?

Reply:
I am testing that there is an item created and that the json response returns the new instance. I am testing it by looking for the new id.